custom_components/unifiprotect/__init__.py

```diff
@@ -136,6 +140,20 @@ def setup(hass, config):
         _LOGGER.error("Unable to connect to NVR: %s", str(ex))
         raise PlatformNotReady
 
```

Comment:
This is the preferred way by HA to fetch data when it's used for many entities: https://developers.home-assistant.io/docs/integration_fetching_data#coordinated-single-api-poll-for-data-for-all-entities